GNUmakefile

```diff
@@ -182,12 +182,17 @@ test-race:
 
 # Run tests with config for CI so `make test` can still be local-dev friendly.
 test-ci: other-consul dev-build vet test-install-deps
-	@ if ! GOTEST_FLAGS="-p 3 -parallel 1" make test-internal; then \
+	@ if ! GOTEST_FLAGS="-short -timeout 8m -p 3 -parallel 4" make test-internal; then \
```

Comment:
We also need to go back to `-parallel 1` here.

Reply:
@banks Actually, keeping parallel helps a bit here:
 - in the first pass, we try to pass as much tests as we can as Travis is so sloooow => parallel does detects failures or parallelisation if any and using -short do not run test cases which are too slow
 - in the next phases, we use the fact that go does cache the result of successful tests, but this time, we completely disable parallel.

Once all tests have been fixed one by one, we will probably be able to let parallel = 4 + -p 4 for instance in order to speed up tests